docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,42 @@
+# Initial deployment
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom with
+the internal programmer.
+
+## Deploy using Dasharo Tools Suite
+
+For simplicity we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md) to
+omit all compilation steps and deploy the Dasharo firmware seamlessly.
+
+1. Boot Dasharo Tools Suite.
+2. Perform Dasharo installation.
+
+After the successful operation, DTS will reboot the platform.
+
+This concludes Dasharo deployment process using DTS.
+
+## Build flashrom
+
+Please follow generic guide for [Dasharo flashrom fork](../../osf-trivia-list/deployment.md#how-to-install-dasharo-flashrom-fork).
+
+## Reading flash contents
+
+Always prepare a backup of the current firmware image. To read from the flash
+and save it to a file (`dump.rom`), execute the following command:
+
+```bash
+flashrom -p internal -r dump.rom
+```
```

Comment:
This should be done using the DTS HCL Report.

Reply:
Yes, but if someone wants manual process, then it is needed.

Reply:
https://github.com/Dasharo/docs/pull/794/commits/e59c85b042692ab32d4c97c05a3c2367d9eb1534